pkgs/os-specific/linux/libnvme/default.nix

```diff
@@ -35,6 +35,11 @@ stdenv.mkDerivation (finalAttrs: {
       url = "https://github.com/linux-nvme/libnvme/commit/ff742e792725c316ba6de0800188bf36751bd1d1.patch";
       hash = "sha256-IUjPUBmGQC4oAKFFlBrjonqD2YdyNPC9siK4t/t2slE=";
     })
+    # included in next release
+    (fetchpatch {
+      url = "https://github.com/linux-nvme/libnvme/pull/727.patch";
```

Comment:
@pbsds, do you know how to rewrite this to include the commit hash(es)?

Reply:
Instead of fetching the generated PR patch from github, fetch the individual commits that were merged to libnvme via that PR:
- https://github.com/linux-nvme/libnvme/commit/a2b8e52e46cfd888ac5a48d8ce632bd70a5caa93.patch
- https://github.com/linux-nvme/libnvme/commit/68c6ffb11d40a427fc1fd70ac2ac97fd01952913.patch

(in this order)